vendor/modules.txt

```diff
@@ -2048,9 +2051,11 @@ k8s.io/client-go/util/retry
 # k8s.io/klog v1.0.0
 ## explicit
 k8s.io/klog
+# k8s.io/klog/v2 v2.3.0
+k8s.io/klog/v2
```

Comment:
We currently try to silent any klog outputs because the clients see https://github.com/openshift/installer/blob/88a611e858a414910e69edeaacdfa695569ce7f9/cmd/openshift-install/main.go#L27-L32

Adding klog/v2 could be reverting that change. We need to make sure no log output ever comes out of klog.

Reply:
makes sense.  I'm happy to idle this PR until the klog/v2 pivot happens, since that seems like an orthogonal change that the installer should pick up in a targeted PR.

Reply:
https://github.com/openshift/installer/pull/4245#issuecomment-714814915